aspnetcore/security/authorization/claims.md

```diff
@@ -22,112 +22,46 @@ An identity can contain multiple claims with multiple values and can contain mul
 
 ## Adding claims checks
 
-Claim based authorization checks are declarative - the developer embeds them within their code, against a controller or an action within a controller, specifying claims which the current user must possess, and optionally the value the claim must hold to access the requested resource. Claims requirements are policy based, the developer must build and register a policy expressing the claims requirements.
+Claim based authorization checks:
 
-The simplest type of claim policy looks for the presence of a claim and doesn't check the value.
+* Are declarative.
+* Are applied to Razor Pages, controllers, or actions within a controller. 
+* can ***not*** be applied at the Razor Page handler level, they must be applied to the Page.
 
-Build and register the policy. This takes place as part of the Authorization service configuration, which normally takes part in `ConfigureServices()` in your *Startup.cs* file.
+Claims in code specify claims which the current user must possess, and optionally the value the claim must hold to access the requested resource. Claims requirements are policy based, the developer must build and register a policy expressing the claims requirements.
```

Comment:
I had to rewrite this. Long sentences don't MT (Machine Translate) and are difficult for native readers to parse.